website/src/components/discourse/index.js

```diff
@@ -0,0 +1,194 @@
+
+  // Set initial min-height
+  // This is to avoid layout shifts
+  // which affects Lighthouse performance scores
+  const setMinHeight = isError
+    ? 'auto'
+    : 414
+
+  return (
+    <div className={feedStyles.discourseTopics} style={{minHeight: setMinHeight, ...styles}}>
+      {title && (
+        <h2>{title}</h2>
+      )}
+      {loading ? (
+        <img src="/img/loader-icon.svg" alt="Loading" className={feedStyles.loadingIcon} />
+      ) : isError || !topics?.length > 0 ? (
+        <p>Unable to load forum topics at this time.</p>
+      ) : (
+        <ul>
+          {topics.map(topic => (
+            <li key={topic.id}>
+              {topic?.has_accepted_answer && (
+                <span className={feedStyles.solvedTopic} title="Solved">✅ </span>
+              )}
+              <TopicWrapper topic={topic}>{topic.title}</TopicWrapper>
+              {(topic?.author || topic?.posts_count || topic?.like_count) && (
+                <>
+                  {' '}
+                  <span>
+                    {topic?.author && `by ${topic.author}${topic?.posts_count ? ',' : ''}`}
+                    {' '}
+                    {topic?.posts_count && `${topic.posts_count - 1} ${(topic.posts_count - 1) === 1 ? 'reply' : 'replies'}${topic?.like_count ? ',' : ''}`}
+                    {' '}
+                    {topic?.like_count ? `${topic.like_count} ${(topic.like_count) === 1 ? 'like' : 'likes'}` : ''}
+                  </span>
+                </>
+              )}
+            </li>
+          ))}
+        </ul>
+      )}
+      {show_cta && (
+        <a className={`button button--primary ${feedStyles.discourseCta}`} href={link_href} title={link_text} target="_blank">{link_text}</a>
+      )}
+    </div>
+  )
+}
+
+// A more specific version of DiscourseFeed
+// with default props set. Same props can be used
+export const DiscourseHelpFeed = ({
+  order = 'latest_topic',
+  status = 'solved',
+  category = 'help',
+  tags, 
+  show_cta = true,
+  link_text = 'Ask the Community',
+  link_href = `https://discourse.getdbt.com/new-topic${category ? `?category=${category}` : ''}${tags ? (!category ? `?tags=${tags}` : `&tags=${tags}`) : ''}`,
+  after,
+  before,
+  inString, 
+  min_posts,
+  max_posts,
+  min_views,
+  max_views,
+  term,
+  title,
+  topic_count = 5,
+  styles = {}
+}) => {
+  return <DiscourseFeed 
+    order={order}
+    status={status}
+    category={category}
+    link_text={link_text}
+    link_href={link_href}
+    show_cta={show_cta}
+    after={after}
+    before={before}
+    inString={inString}
+    min_posts={min_posts}
+    max_posts={max_posts}
+    min_views={min_views}
+    max_views={max_views}
+    term={term}
+    title={title}
+    topic_count={topic_count}
+    styles={styles}
+  />
+}
+
+// Helpers
+function TopicWrapper({ topic, children }) {
+  if(topic?.slug) {
+    return (
+      <a href={`https://discourse.getdbt.com/t/${topic.slug}`} title={topic.title} target="_blank">{children}</a>
```

Comment:
The slug is human-readable, but the actual important part of the link is the topic ID. 

https://discourse.getdbt.com/t/compare-two-models-schemas will redirect to https://discourse.getdbt.com/t/compare-two-models-schemas/3868

https://discourse.getdbt.com/t/fjhsbjhfbshjf/3868 will also get to the right place. 

I don't think this is _likely_ to break since we're using search results from a few seconds ago and the slug probably won't have been changed by a topic rename, but it's worth doing it the proper-er way.

Reply:
(and _THEN_ I promise I'm done with feedback 😅)

Reply:
good call, just updated!